db/migrate/20250807063339_create_join_table_schools_lead_providers_contract_periods_metadata.rb

```diff
@@ -0,0 +1,14 @@
+class CreateJoinTableSchoolsLeadProvidersContractPeriodsMetadata < ActiveRecord::Migration[8.0]
+  def change
+    create_table :schools_lead_providers_contract_periods_metadata do |t|
```

Comment:
Might it be clearer from a database point of view if they were prefixed with `metadata` rather than suffixed?

Reply:
Yeah that's a good shout, it follows the namespace that way as well. I'll update it 👍